frontend/device/kobo/device.lua

```diff
@@ -424,6 +425,10 @@ local KoboIo = Kobo:new{
     -- It would appear that the Libra 2 inherited its ancestor's quirks, and more...
     -- c.f., https://github.com/koreader/koreader/issues/8414 & https://github.com/koreader/koreader/issues/8664
     hasReliableMxcWaitFor = no,
+    -- NOTE: There are at least two hardware revisions of this device (*without* a device code change, this time),
+    --       with *significant* hardware changes, so we'll handle this by making the sysfs path discovery automagic.
+    --       c.f., https://github.com/koreader/koreader/issues/9218
+    automagic_sysfs = true,
```

Comment:
Didn't you once ask me to change automagic when I used it in a vaguely similar context? ;-P

Reply:
That... doesn't ring a bell ;).

(as far as I'm concerned, usage stems from deep psychological scarring left by dealing with autotools automagic feature checks ;p).

Reply:
Shouldn't it be defined as `automagic_sysfs = false,` in `local Kobo = Generic:new{}` for consistency/documentation ?